src/main.rs

```diff
@@ -817,6 +817,9 @@ fn set_hstateen(cpu_info: &CpuInfo) {
     }
     // Svadu is a required feature for Salus.
     CSR.henvcfg.modify(henvcfg::adue.val(1));
+    if cpu_info.has_svpbmt() {
+        CSR.henvcfg.modify(henvcfg::pbmte.val(1));
+    }
```

Comment:
move this and adue to set_henvcfg instead of set_hstateen ?

Reply:
yes, that was a careless rebase...